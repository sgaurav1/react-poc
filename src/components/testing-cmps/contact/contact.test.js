import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Contact from './contact';

const phone = "8888888888";

describe('<Contact />', () => {
    test('it should mount', () => {
        render(<Contact />);

        const profile = screen.getByTestId('Contact');

        expect(profile).toBeInTheDocument();
    });

});

describe('The contact should render correclty', () => {
    test('it shoudl have number', ()=> {
        render(<Contact phone={phone} />);
    const profile = screen.getByTestId('Contact');
    expect(profile).toMatchSnapshot();
})
})