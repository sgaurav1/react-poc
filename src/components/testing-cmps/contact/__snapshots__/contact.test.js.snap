// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The contact should render correclty it shoudl have number 1`] = `
<div
  class="container"
  data-testid="Contact"
>
  <div
    class="shadow p-3"
  >
    <h5
      class="text-success text-center"
      id="phone8888888888"
    >
      Conatcts 
      8888888888
    </h5>
  </div>
</div>
`;
